lib/scope.js

```diff
@@ -2170,10 +2273,244 @@ module.exports = {
       });
 
       await scope.afterStep(scope, {waitForShowIf: false});
-    },  // Ends sign_in
+    },  // Ends scope.steps.sign_in()
+
+    set_random_page_vars: async ( scope ) => {
+      /** Answer inputs randomly. */
+
+      let html = await scope.page.content();
+      // These are the fields on the current page. Their handles should all exist.
+      let fields = await scope.getAllFields( scope, { html: html });
+      let buttons = [];
+
+      // For every field on the page
+      for ( let field of fields ) {
+
+        // * @param fields - data representing one DOM field
+        // * @param fields[n].selector {str} - used to find the DOM node
+        // * @param fields[n].tag {str} - HTML tag of node
+        // * @param fields[n].type {str} - HTML type of node if it has one
+        // * @param fields[n].guesses {arr} - arr of objects that could
+        // *    possibly match a table row's `var` column
+        // * @param fields[n].guesses[n].var {str} - name of variable
+        // * @param fields[n].guesses[n].value {str} - value of a choice
+        // * @param <fields[n].guesses[n].trigger> {str} - var name that triggered this page
```

Comment:
Probably useful for writing the code, but doesn't need to be here.  You could leave the `field[n].tag` explanation if you wanted, since it's the only attribute used here.
```suggestion
        // fields[n].tag is HTML tag of node, as a string
```